libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/RetrofitApiClient.java

```diff
@@ -132,7 +132,7 @@ public String getJourneyXml(final String plan,
                               final String arriving,
                               final int speed) throws IOException {
     Response<String> response = v1Api.getJourneyXml(plan, itineraryPoints, leaving, arriving, speed).execute();
-    return JourneyStringTransformer.fromV1ApiXml(response.body());
+    return JourneyStringTransformerKt.fromV1ApiXml(response.body());
```

Comment:
@jezhiggins I went one better than the companion object idea, and just made [package-level functions](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#package-level-functions) in this case.

Reply:
Ah, good one - because all the methods on JourneyStringTransformer are static, we can do away with the class definition entirely. Nice.